context/Dockerfile

```diff
@@ -1,10 +1,19 @@
+FROM node:12.16.3-alpine as builder
+WORKDIR /work
+COPY . .
+RUN npm install
+RUN npm run prepublish
```

Comment:
I started #232 but didn't finish. Let's just add a TODO here since it should be a quick fix.
```suggestion
# TODO: Get rid of prepublish... wget the prov-vis schema shouldn't be necessary. See PR #232.
RUN npm run prepublish
```